packages/typespec-ts/src/modular/buildCodeModel.ts

```diff
@@ -1342,6 +1343,7 @@ function getNonNullOptions(type: Union) {
 
 function emitEnumMember(type: any): Record<string, any> {
   return {
+    type: enumMemberType(type),
```

Comment:
Could you add a UT? What is the output type for enum member type?

Reply:
I use constant, and we will put the value into value, getType will build the output correctly for us.

Reply:
Could you add a UT in `transformSchemas.spec.ts` in RLC and  typeHelpers.spec.ts in Modular? So we could know their schema details.